fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -29,12 +29,16 @@
 import com.github.dedis.popstellar.ui.wallet.SeedWalletFragment;
 import com.github.dedis.popstellar.ui.wallet.WalletFragment;
 import com.github.dedis.popstellar.utility.ActivityUtils;
+import com.github.dedis.popstellar.utility.error.ErrorUtils;
+import com.github.dedis.popstellar.utility.error.keys.UninitializedWalletException;
 
 import java.util.function.Supplier;
 
 import dagger.hilt.android.AndroidEntryPoint;
 
-/** HomeActivity represents the entry point for the application. */
+/**
```

Comment:
It seems to me like the formatting is wrong.

This kind of one line comments are not allowed in the google-java-format.

Do you have the plugin installed and enabled ?

(It is documented in the README under code-style)

Reply:
I did follow the README and made a small mistake. I fixed it.